scripts/generate-a3p-submission.sh

```diff
@@ -3,17 +3,22 @@ set -ueo pipefail
 
 sdkroot=$(cd -- "$(dirname "$0")/.." >/dev/null && pwd)
 
-cd $sdkroot
+cd "$sdkroot"
 
 buildSubmission() {
-  proposalName=$1
-  a3pProposal=$2
+  proposalName=$1; shift
+  a3pProposal=$1; shift
+  output=${1:-$proposalName}; shift
 
-  yarn agoric run packages/builders/scripts/vats/$1.js
+  # We always pass the `--run-test` flag; it's up to the builder to decide
```

Comment:
per points above, I think we should avoid mixing testing logic with the submissions.